2022-09-19 00:34:32,480 - fishTrade_main20220913.py[line:554] - INFO: 
                  ET网格智能交易系统3.0             
*************************版权声明***************************             
********************All Rights Reserved*********************             
软件开发：海易，需求订制联系微信：yyy99966             
优惠版：有效期30天，每分享一个好友，延长30天，10个延长一年

2022-09-19 00:34:32,481 - fishTrade_main20220913.py[line:142] - INFO: 初始参数设置:symbol=usdt_eth highPrice=1700.0 lowPrice=1500.0  step_num=70.0
2022-09-19 00:34:33,954 - fishTrade_main20220913.py[line:606] - ERROR: 'Cannot connect to proxy.', RemoteDisconnected('Remote end closed connection without response')
2022-09-19 00:34:35,211 - fishTrade_main20220913.py[line:606] - ERROR: 'Cannot connect to proxy.', RemoteDisconnected('Remote end closed connection without response')
2022-09-19 00:34:36,602 - fishTrade_main20220913.py[line:606] - ERROR: 'Cannot connect to proxy.', RemoteDisconnected('Remote end closed connection without response')
2022-09-19 00:34:38,353 - fishTrade_main20220913.py[line:606] - ERROR: 'Cannot connect to proxy.', RemoteDisconnected('Remote end closed connection without response')
2022-09-19 00:34:39,735 - fishTrade_main20220913.py[line:606] - ERROR: 'Cannot connect to proxy.', RemoteDisconnected('Remote end closed connection without response')
2022-09-19 00:34:40,983 - fishTrade_main20220913.py[line:606] - ERROR: 'Cannot connect to proxy.', RemoteDisconnected('Remote end closed connection without response')
2022-09-19 00:34:42,423 - fishTrade_main20220913.py[line:606] - ERROR: 'Cannot connect to proxy.', RemoteDisconnected('Remote end closed connection without response')
2022-09-19 00:34:43,935 - fishTrade_main20220913.py[line:606] - ERROR: 'Cannot connect to proxy.', RemoteDisconnected('Remote end closed connection without response')
2022-09-19 00:34:45,651 - fishTrade_main20220913.py[line:606] - ERROR: 'Cannot connect to proxy.', RemoteDisconnected('Remote end closed connection without response')
2022-09-19 00:34:47,047 - fishTrade_main20220913.py[line:288] - ERROR: Traceback (most recent call last):
  File "E:\GitHub_rep\test_gateAPI\fishTrade_main20220913.py", line 165, in strategy
    print(huobi_market.market_detail(self.quote_cur,self.base_cur))
  File "E:\GitHub_rep\test_gateAPI\marketHelper1005.py", line 386, in market_detail
    r = requests.request('GET', acfg.gateApi + prefix + url + "?" + query_param, headers=headers)
  File "D:\Python\lib\site-packages\requests\api.py", line 59, in request
    return session.request(method=method, url=url, **kwargs)
  File "D:\Python\lib\site-packages\requests\sessions.py", line 587, in request
    resp = self.send(prep, **send_kwargs)
  File "D:\Python\lib\site-packages\requests\sessions.py", line 701, in send
    r = adapter.send(request, **kwargs)
  File "D:\Python\lib\site-packages\requests\adapters.py", line 489, in send
    resp = conn.urlopen(
  File "D:\Python\lib\site-packages\urllib3\connectionpool.py", line 703, in urlopen
    httplib_response = self._make_request(
  File "D:\Python\lib\site-packages\urllib3\connectionpool.py", line 386, in _make_request
    self._validate_conn(conn)
  File "D:\Python\lib\site-packages\urllib3\connectionpool.py", line 1042, in _validate_conn
    conn.connect()
  File "D:\Python\lib\site-packages\urllib3\connection.py", line 414, in connect
    self.sock = ssl_wrap_socket(
  File "D:\Python\lib\site-packages\urllib3\util\ssl_.py", line 449, in ssl_wrap_socket
    ssl_sock = _ssl_wrap_socket_impl(
  File "D:\Python\lib\site-packages\urllib3\util\ssl_.py", line 493, in _ssl_wrap_socket_impl
    return ssl_context.wrap_socket(sock, server_hostname=server_hostname)
  File "D:\Python\lib\ssl.py", line 513, in wrap_socket
    return self.sslsocket_class._create(
  File "D:\Python\lib\ssl.py", line 1071, in _create
    self.do_handshake()
  File "D:\Python\lib\ssl.py", line 1342, in do_handshake
    self._sslobj.do_handshake()
KeyboardInterrupt

2022-09-19 00:34:47,049 - fishTrade_main20220913.py[line:606] - ERROR: 'Cannot connect to proxy.', RemoteDisconnected('Remote end closed connection without response')
2022-09-19 00:34:49,025 - fishTrade_main20220913.py[line:606] - ERROR: 'Cannot connect to proxy.', RemoteDisconnected('Remote end closed connection without response')
2022-09-19 00:34:51,184 - fishTrade_main20220913.py[line:606] - ERROR: 'Cannot connect to proxy.', RemoteDisconnected('Remote end closed connection without response')
2022-09-19 00:34:52,372 - fishTrade_main20220913.py[line:606] - ERROR: 'Cannot connect to proxy.', RemoteDisconnected('Remote end closed connection without response')
2022-09-19 00:34:54,665 - fishTrade_main20220913.py[line:606] - ERROR: 'Cannot connect to proxy.', RemoteDisconnected('Remote end closed connection without response')
2022-09-19 00:34:56,206 - fishTrade_main20220913.py[line:606] - ERROR: 'Cannot connect to proxy.', RemoteDisconnected('Remote end closed connection without response')
2022-09-19 00:34:59,020 - fishTrade_main20220913.py[line:606] - ERROR: 'Cannot connect to proxy.', RemoteDisconnected('Remote end closed connection without response')
2022-09-19 00:35:00,267 - fishTrade_main20220913.py[line:606] - ERROR: 'Cannot connect to proxy.', RemoteDisconnected('Remote end closed connection without response')
2022-09-19 00:35:01,777 - fishTrade_main20220913.py[line:606] - ERROR: 'Cannot connect to proxy.', RemoteDisconnected('Remote end closed connection without response')
2022-09-19 00:35:03,574 - fishTrade_main20220913.py[line:606] - ERROR: 'Cannot connect to proxy.', RemoteDisconnected('Remote end closed connection without response')
2022-09-19 00:35:07,416 - fishTrade_main20220913.py[line:606] - ERROR: 'Cannot connect to proxy.', RemoteDisconnected('Remote end closed connection without response')
